src/structs/context.rs

```diff
@@ -131,6 +131,21 @@ context_methods! {
         crate::say_reply(self, text).await
     }
 
+    /// Shorthand of [`crate::reply_reply`]
+    ///
+    /// reply to the reference message in prefix commands
+    ///
+    /// for slash commands, this is the same as [`say`].
+    ///
+    /// Note: panics when called in an autocomplete context!
+    await (say self text)
```

Comment:
Please change the method name here from say to reply, or the method implementations on PrefixContext and ApplicationContext will have the wrong name